inject.js

```diff
@@ -321,11 +325,18 @@ chrome.extension.sendMessage({}, function(response) {
           controller.classList.toggle('vsc-hidden');
         } else if (action === 'drag') {
           handleDrag(v, controller);
+        } else if (action === 'fast') {
+          playVideoAtFastSpeed(v);
         }
       }
     });
   }
 
+  // CONTRACT: video has a way to set the playback rate.
```

Comment:
Gotcha. Yeah, let's drop it, it's implicit here and stands out as a red flag with respect to the rest of the file.